content/test/integration/controller/active-entity-by-content-hash.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { EnvironmentConfig } from '@katalyst/content/Environment'
+import { EntityId, Fetcher } from 'dcl-catalyst-commons'
+import fetch from 'node-fetch'
+import { loadStandaloneTestEnvironment } from '../E2ETestEnvironment'
+import { buildDeployData } from '../E2ETestUtils'
+import { TestServer } from '../TestServer'
+
+describe('Integration - Get Active Entity By Content Hash', () => {
```

Comment:
```suggestion
describe('Integration - Get Active Entities By Content Hash', () => {
```

We should probably also update the test file name